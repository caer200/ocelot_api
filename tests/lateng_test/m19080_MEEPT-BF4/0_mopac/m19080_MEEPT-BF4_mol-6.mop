CHARGES

 
H	3.604816 +1 4.837341 +1 15.674388 +1
C	3.129231 +1 4.775713 +1 14.854297 +1
C	3.818839 +1 4.958215 +1 13.631055 +1
C	1.784236 +1 4.510812 +1 14.870158 +1
C	3.073968 +1 4.782523 +1 12.441718 +1
N	5.171393 +1 5.302723 +1 13.619868 +1
H	1.342768 +1 4.391109 +1 15.702791 +1
C	1.050704 +1 4.414112 +1 13.686007 +1
C	1.692404 +1 4.534646 +1 12.481092 +1
S	3.763065 +1 4.857090 +1 10.867210 +1
C	5.968257 +1 5.330711 +1 12.481682 +1
C	5.793071 +1 5.757686 +1 14.891955 +1
H	0.112783 +1 4.266367 +1 13.715296 +1
H	1.202130 +1 4.449782 +1 11.671894 +1
C	5.434631 +1 5.109393 +1 11.186281 +1
C	7.354623 +1 5.608551 +1 12.569023 +1
H	6.411869 +1 6.504244 +1 14.691422 +1
H	5.081195 +1 6.110692 +1 15.482591 +1
C	6.557350 +1 4.685142 +1 15.643490 +1
C	6.252640 +1 5.164552 +1 10.047121 +1
H	7.755925 +1 5.737365 +1 13.420441 +1
C	8.125336 +1 5.695036 +1 11.432585 +1
H	7.075909 +1 5.094248 +1 16.380873 +1
H	7.190510 +1 4.231344 +1 15.032448 +1
O	5.634536 +1 3.744981 +1 16.166617 +1
H	5.882768 +1 4.985788 +1 9.190610 +1
C	7.583641 +1 5.475079 +1 10.164409 +1
H	9.047517 +1 5.908843 +1 11.512389 +1
C	6.284204 +1 2.768184 +1 16.991893 +1
H	8.131220 +1 5.540235 +1 9.390820 +1
H	6.802681 +1 2.141035 +1 16.428006 +1
H	6.909222 +1 3.215438 +1 17.615863 +1
C	5.241774 +1 2.019106 +1 17.762751 +1
H	4.677564 +1 2.655985 +1 18.269155 +1
H	5.676989 +1 1.408859 +1 18.409152 +1
O	4.433878 +1 1.270913 +1 16.862407 +1
C	3.408200 +1 0.559085 +1 17.538638 +1
H	2.875830 +1 1.183223 +1 18.074496 +1
H	2.830302 +1 0.116502 +1 16.882398 +1
H	3.810759 +1 -0.114589 +1 18.125929 +1
