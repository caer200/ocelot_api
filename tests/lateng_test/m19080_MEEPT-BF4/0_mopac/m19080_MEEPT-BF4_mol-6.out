 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:31:06 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          3.60481600  *   4.83734100  *  15.67438800  *
     2       C          3.12923100  *   4.77571300  *  14.85429700  *
     3       C          3.81883900  *   4.95821500  *  13.63105500  *
     4       C          1.78423600  *   4.51081200  *  14.87015800  *
     5       C          3.07396800  *   4.78252300  *  12.44171800  *
     6       N          5.17139300  *   5.30272300  *  13.61986800  *
     7       H          1.34276800  *   4.39110900  *  15.70279100  *
     8       C          1.05070400  *   4.41411200  *  13.68600700  *
     9       C          1.69240400  *   4.53464600  *  12.48109200  *
    10       S          3.76306500  *   4.85709000  *  10.86721000  *
    11       C          5.96825700  *   5.33071100  *  12.48168200  *
    12       C          5.79307100  *   5.75768600  *  14.89195500  *
    13       H          0.11278300  *   4.26636700  *  13.71529600  *
    14       H          1.20213000  *   4.44978200  *  11.67189400  *
    15       C          5.43463100  *   5.10939300  *  11.18628100  *
    16       C          7.35462300  *   5.60855100  *  12.56902300  *
    17       H          6.41186900  *   6.50424400  *  14.69142200  *
    18       H          5.08119500  *   6.11069200  *  15.48259100  *
    19       C          6.55735000  *   4.68514200  *  15.64349000  *
    20       C          6.25264000  *   5.16455200  *  10.04712100  *
    21       H          7.75592500  *   5.73736500  *  13.42044100  *
    22       C          8.12533600  *   5.69503600  *  11.43258500  *
    23       H          7.07590900  *   5.09424800  *  16.38087300  *
    24       H          7.19051000  *   4.23134400  *  15.03244800  *
    25       O          5.63453600  *   3.74498100  *  16.16661700  *
    26       H          5.88276800  *   4.98578800  *   9.19061000  *
    27       C          7.58364100  *   5.47507900  *  10.16440900  *
    28       H          9.04751700  *   5.90884300  *  11.51238900  *
    29       C          6.28420400  *   2.76818400  *  16.99189300  *
    30       H          8.13122000  *   5.54023500  *   9.39082000  *
    31       H          6.80268100  *   2.14103500  *  16.42800600  *
    32       H          6.90922200  *   3.21543800  *  17.61586300  *
    33       C          5.24177400  *   2.01910600  *  17.76275100  *
    34       H          4.67756400  *   2.65598500  *  18.26915500  *
    35       H          5.67698900  *   1.40885900  *  18.40915200  *
    36       O          4.43387800  *   1.27091300  *  16.86240700  *
    37       C          3.40820000  *   0.55908500  *  17.53863800  *
    38       H          2.87583000  *   1.18322300  *  18.07449600  *
    39       H          2.83030200  *   0.11650200  *  16.88239800  *
    40       H          3.81075900  *  -0.11458900  *  18.12592900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          3.6048      4.8373     15.6744
     2         C          3.1292      4.7757     14.8543
     3         C          3.8188      4.9582     13.6311
     4         C          1.7842      4.5108     14.8702
     5         C          3.0740      4.7825     12.4417
     6         N          5.1714      5.3027     13.6199
     7         H          1.3428      4.3911     15.7028
     8         C          1.0507      4.4141     13.6860
     9         C          1.6924      4.5346     12.4811
    10         S          3.7631      4.8571     10.8672
    11         C          5.9683      5.3307     12.4817
    12         C          5.7931      5.7577     14.8920
    13         H          0.1128      4.2664     13.7153
    14         H          1.2021      4.4498     11.6719
    15         C          5.4346      5.1094     11.1863
    16         C          7.3546      5.6086     12.5690
    17         H          6.4119      6.5042     14.6914
    18         H          5.0812      6.1107     15.4826
    19         C          6.5573      4.6851     15.6435
    20         C          6.2526      5.1646     10.0471
    21         H          7.7559      5.7374     13.4204
    22         C          8.1253      5.6950     11.4326
    23         H          7.0759      5.0942     16.3809
    24         H          7.1905      4.2313     15.0324
    25         O          5.6345      3.7450     16.1666
    26         H          5.8828      4.9858      9.1906
    27         C          7.5836      5.4751     10.1644
    28         H          9.0475      5.9088     11.5124
    29         C          6.2842      2.7682     16.9919
    30         H          8.1312      5.5402      9.3908
    31         H          6.8027      2.1410     16.4280
    32         H          6.9092      3.2154     17.6159
    33         C          5.2418      2.0191     17.7628
    34         H          4.6776      2.6560     18.2692
    35         H          5.6770      1.4089     18.4092
    36         O          4.4339      1.2709     16.8624
    37         C          3.4082      0.5591     17.5386
    38         H          2.8758      1.1832     18.0745
    39         H          2.8303      0.1165     16.8824
    40         H          3.8108     -0.1146     18.1259

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
